cellxgene_schema_cli/cellxgene_schema/schema_definitions/3_0_0.yaml

```diff
@@ -105,6 +117,9 @@ components:
         required: null # Means it's required
         index:
             unique: true
+        # deprecated_columns:
```

Comment:
Needs to be uncommented when https://github.com/chanzuckerberg/single-cell-curation/pull/305 is merged